crypto/rand/rand_unix.c

```diff
@@ -201,14 +188,25 @@ static size_t sysctl_random(char *buf, size_t buflen)
     size_t len;
 
     /*
-     * Old implementations returned longs, newer versions support variable
-     * sizes up to 256 byte. The code below would not work properly when
-     * the sysctl returns long and we want to request something not a multiple
-     * of longs, which should never be the case.
+     * On FreeBSD old implementations returned longs, newer versions support
+     * variable sizes up to 256 byte. The code below would not work properly
+     * when the sysctl returns long and we want to request something not a
+     * multiple of longs, which should never be the case.
      */
     if (!ossl_assert(buflen % sizeof(long) == 0))
         return 0;
 
+    /*
+     * On NetBSD before 4.0 KERN_ARND was an alias for KERN_URND, and only
```

Comment:
4.0 is no longer supported.. 5.x is EOL and 6.0 will be soon EOL and 8.0 should be released soon.

Reply:
People tend to use systems much longer than they are supported. But you're right, this really is a check for an obsolete system, and we could probably perfectly live without it. But it's mostly there for documentation purpose.

Anyway, it would be nice that NetBSD also added support for getentropy().

Reply:
Please propose it to other developers on a mailinglist. Personally I have no opinion.